.github/scripts/generate_table_versions.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+# Input and output file names
+input_file = 'gem_output.json'
+output_file = 'integration_versions.md'
+
+# Read JSON data from the input file
+data = JSON.parse(File.read(input_file))
+
+# Prepare the Markdown content
+comment = "# This is a table of supported integration versions generated from gemfiles.\n\n"
```

Comment:
```suggestion
comment = "# Integrations\n\n"
```

nitpicky, but maps to the current [Ruby compatiblity](https://docs.datadoghq.com/tracing/trace_collection/compatibility/ruby/#integrations)